config/index.js

```diff
@@ -17,6 +17,7 @@ const SESSION = {
   TTL: process.env.SESSION_TTL || 60 * 30 * 1000, // 30 mins
   DB: process.env.SESSION_DB_INDEX || 0,
 }
+const AUTH_EXPIRY_MARGIN = process.env.AUTH_EXPIRY_MARGIN || 5 * 60 // 5 minutes
```

Comment:
This could be done inline? I think the other process envs defined here are because they're used more than once.

Reply:
Sure